package.json

```diff
@@ -13,10 +13,11 @@
     "bootstrap": "lerna bootstrap",
     "setup": "yarn install && yarn run bootstrap",
     "start": "lerna run start --stream",
+    "start:op": "POSTGRES_URL=postgres://localhost/opportunities lerna run start --stream",
```

Comment:
my local is set to postgresql://localhost:5432/cares_grant. Im fine with moving to this url and standardizing